### DISCUSSION TITLE
How to redirect .get('/')

### DISCUSSION THREAD

**Discussion by cevr** [fastify/fastify#3486] [Q&A]
Im trying to redirect the '/' route to somewhere else but the route is never hit. No matter what I do it just ignores that handler. Is there something I'm missing?

**Comment by gregalia** [comment]
Hello @cevr,

Please post a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).

**Reply by cevr** [reply]
Yeah I should have given more of an effort to help with this. The issue ended up being that it was called for routes like `127.0.0.1:3000`but not for `localhost:3000`. In production it wasnt an issue. Not sure what that means, maybe its intended?
